packages/react/src/components/Tooltip/Tooltip.tsx

```diff
@@ -81,6 +83,8 @@ export interface TooltipProps
 /**
  * A tooltip that displays information related to an element when the element receives keyboard focus
  * or the mouse hovers over it.
+ * @accessibility
+ * If adding tooltip to icon-only button or to another visual-only element without any text, title or label, apply `tooltipAsLabelBehavior` to `accessibility` prop to correctly label the element for all screen readers.
```

Comment:
please move this information to the Best Practices section of the tooltip site - it is more visible there.